radio/src/targets/horus/CMakeLists.txt

```diff
@@ -162,21 +151,26 @@ elseif (PCB STREQUAL X12S)
   set(BLUETOOTH ON)
   set(PCBREV "13" CACHE STRING "PCB Revision")
   if(${PCBREV} GREATER 10)
```

Comment:
Drop completely, not need to check PCBREV with this change

Reply:
Wouldn't the change I made enable INTERNAL_GPS for PCBREV less than 10 for X12S? Or is this in reference to the fact you can assign/configure it at runtime, so not so much an issue anymore?

Reply:
Since you enable it for all radio with a serial port, you shouldn't make that difference here to stay in the same spirit

Reply:
Sure, but this is what the "unless otherwise disabled" also partly related to... so should it be a complete drop, or rather just the disable needed?

i.e. 

```cpp
if(${PCBREV} LESS 10)
  option(INTERNAL_GPS "Support for internal GPS" NO)
endif()
```

Reply:
When I read the old code, it seems to me that greater than 10 get it no matter what, lower don't default it, but a message says you can enable it, so I assume it is safe both ways